From a82d52de3ed05bf65334694e083d8ff50aa82d2c Mon Sep 17 00:00:00 2001
From: Olliver Schinagl <oliver@schinagl.nl>
Date: Wed, 11 Dec 2019 17:14:24 +0100
Subject: [PATCH 2/2] Add: Accept an argument

Signed-off-by: Olliver Schinagl <oliver@schinagl.nl>
---
 hello_world.sh | 9 ++++++++-
 1 file changed, 8 insertions(+), 1 deletion(-)

diff --git a/hello_world.sh b/hello_world.sh
index fd9f355..8bc9fa0 100755
--- a/hello_world.sh
+++ b/hello_world.sh
@@ -20,6 +20,7 @@ cleanup()
 
 world()
 {
+	${1:+echo "Argument was: '${1}'"}
 	echo "Hello, World!"
 }
 
@@ -43,12 +44,18 @@ main()
 	done
 	shift "${#OPTARG}"
 
+	if [ "${#}" -le 0 ]; then
+		echo "Need at least one argument"
+		usage
+		exit 1
+	fi
+
 	if [ "$(id -u)" -eq 0 ]; then
 		echo "Do not run as root"
 		exit 1
 	fi
 
-	world
+	world "${@}"
 
 	cleanup
 }
-- 
2.20.1

