From 0010aa0253708bc5dc1e849f1b3fa6ff5c4fa09c Mon Sep 17 00:00:00 2001
From: Olliver Schinagl <oliver@schinagl.nl>
Date: Wed, 11 Dec 2019 17:11:12 +0100
Subject: [PATCH 1/2] Bug: Shift processed options

Signed-off-by: Olliver Schinagl <oliver@schinagl.nl>
---
 hello_world.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/hello_world.sh b/hello_world.sh
index 880b5d1..fd9f355 100755
--- a/hello_world.sh
+++ b/hello_world.sh
@@ -41,6 +41,7 @@ main()
 			;;
 		esac
 	done
+	shift "${#OPTARG}"
 
 	if [ "$(id -u)" -eq 0 ]; then
 		echo "Do not run as root"
-- 
2.20.1

